src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -114,11 +249,52 @@ or PlaybackStateCompat.StateSkippingToQueueItem
 		};
 
 		MediaElement.CurrentStateChanged(newState);
-
+		
 		if (playbackState is IPlayer.StateReady)
 		{
 			MediaElement.Duration = TimeSpan.FromMilliseconds(Player.Duration < 0 ? 0 : Player.Duration);
 			MediaElement.Position = TimeSpan.FromMilliseconds(Player.CurrentPosition < 0 ? 0 : Player.CurrentPosition);
+			if (mediaSession is not null)
+			{
+				mediaSession.Active = true;
+			}
+			CancellationToken cancellationToken = new();
```

Comment:
Who can cancel this token? The correct approach is to create a `CancellationTokenSource` and use the `CancellationToken` inside it, that way we can cancel the token. This implementation will not allow cancelling.